wordpress-mobile/WordPress-Android
# [DO NOT MERGE] Editing: Remove snackbar when switching to html or visual mode

Fixes #11819

## Description
We received remarks that the snackbar notice shown when switching to the HTML editor prevents the user from being able to edit the last line(s) of text until it disappears.
This appears to be annoying for some users and doesn't seem to add any value.

On iOS or in the web version there is no such notification and the users can switch seamlessly between the Visual or the HTML editor mode.

It was suggested by @yaelirub to remove it and in a recent conversation suggested also by @antonis.

Given the above context I propose this solution, which:
- Removes the snackbar notice shown when switching to the HTML Mode in the editor, and
- Removes the snackbar notice shown when switching to the Visual Mode

## To test
1. From My Site press the Posts button or scroll down and select Blog Posts from the list
2. In post cards mode press the More button / In post list mode press the dots at the right of the post
3. Press Edit (first button)
4. In the editor, Tap the 3 dots in the upper right corner
5. Tap HTML Mode (3rd button)
6. **Verify** that no snackbar notice appears at the bottom of the screen
7. Tap again the 3 dots in the upper right corner
8. Tap Visual Mode  (3rd button)
9. **Verify** that no snackbar notice appears at the bottom of the screen

## Screenshots
<details>

 <summary>Switching to HTML mode</summary>

| Before | After |
| ----- | ----- |
| ![android_switch_to_html_before](https://user-images.githubusercontent.com/4588074/141684301-a6df03c3-ad4a-47db-a495-8cad238250b3.png) |  ![android_switch_to_html_after](https://user-images.githubusercontent.com/4588074/141690261-bf2b246f-0db6-4879-b065-aadbf3c323af.png) |
</details>

<details>

 <summary>Switching to Visual mode</summary>

| Before | After |
| ----- | ----- |
| ![android_switch_to_visual_before](https://user-images.githubusercontent.com/4588074/141684310-1e2c8ada-34ba-445e-a3d9-f6a2462124af.png) |  ![android_switch_to_visual_after](https://user-images.githubusercontent.com/4588074/141690353-1d2b5b2c-c3bc-4e49-938a-7c5b2ca984a6.png) |
</details>

## Regression Notes
1. Potential unintended areas of impact
  none

2. What I did to test those areas of impact (or what existing automated tests I relied on)
  manual testing

3. What automated tests I added (or what prevented me from doing so)
  n/a

PR submission checklist:
